stages/org.osbuild.chrony

```diff
@@ -54,13 +54,121 @@ def handle_leapsectz(chrony_conf_lines, timezone):
         chrony_conf_lines[:] = [f"leapsectz {timezone}"] + chrony_conf_lines
 
 
+def handle_refclocks(chrony_conf_lines, refclocks):
+    new_lines = []
+
+    for refclock in refclocks:
+        driver = refclock["driver"]
+        driver_name = driver["name"]
+        new_line = "refclock"
```

Comment:
What will happen if "refclock" is already set as part of the pre-existing config? It seems we will write two refclock lines then - is that a problem? Or do we need to deal with this just like we deal with leapsectz (i.e. delete if pre-existing)?

Reply:
Multiple refclocks are supported.  That's why I made it an array.  Existing refclocks shouldn't be overwritten.

Reply:
Example from chrony docs: https://chrony-project.org/examples.html#_server_using_reference_clock_on_nic